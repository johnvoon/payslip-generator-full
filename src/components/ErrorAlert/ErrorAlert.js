import React from "react";
import PropTypes from "prop-types";

const ErrorAlert = ({ message }) => {
  return (
    <div className="alert alert-danger" role="alert">
      <span className="" aria-hidden="true" />
      <span className="sr-only">Error:</span>
      {message}
    </div>
  );
};

ErrorAlert.propTypes = {
  message: PropTypes.string.isRequired
};

export default ErrorAlert;
