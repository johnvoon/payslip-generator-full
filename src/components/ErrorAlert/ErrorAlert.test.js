import React from "react";
import ErrorAlert from "./ErrorAlert";
import { shallow } from "enzyme";

describe("ErrorAlert", () => {
  const renderedComponent = shallow(<ErrorAlert message="Error" />);

  it("should match snapshot", () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
