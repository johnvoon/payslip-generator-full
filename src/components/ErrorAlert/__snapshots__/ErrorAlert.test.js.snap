exports[`ErrorAlert should match snapshot 1`] = `
<div
  className="alert alert-danger"
  role="alert">
  <span
    aria-hidden="true"
    className="" />
  <span
    className="sr-only">
    Error:
  </span>
  Error
</div>
`;
